packages/console/src/pages/SignInExperience/components/Preview.tsx

```diff
@@ -27,6 +27,50 @@ const Preview = ({ signInExperience, className }: Props) => {
   const { data: allConnectors } = useSWR<ConnectorDTO[], RequestError>('/api/connectors');
   const previewRef = useRef<HTMLIFrameElement>(null);
 
+  const modeOptions = useMemo(() => {
+    const light = { value: AppearanceMode.LightMode, title: t('sign_in_exp.preview.light') };
+    const dark = { value: AppearanceMode.DarkMode, title: t('sign_in_exp.preview.dark') };
+
+    if (!signInExperience?.branding.isDarkModeEnabled) {
+      return [light];
+    }
+
+    return [light, dark];
+  }, [signInExperience, t]);
+
+  useEffect(() => {
+    if (!modeOptions[0]) {
+      return;
+    }
+
+    if (!modeOptions.some(({ value }) => value === mode)) {
+      setMode(modeOptions[0].value);
+    }
```

Comment:
Should we set default theme mode to the system preferred one?

Reply:
Is a nice to have feature, let's move on?